client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-tasks.ts

```diff
@@ -1,4 +1,10 @@
 import { translate as i18nTranslate } from 'i18n-calypso';
+import customDomain from 'calypso/assets/images/plans/wpcom/custom-domain.png';
+import customize from 'calypso/assets/images/plans/wpcom/customize.png';
+import manage from 'calypso/assets/images/plans/wpcom/manage.png';
```

Comment:
I can't comment on the image, so I am going to comment here: I think the image name should be changed to `manage-woocommerce.png` or `manage-woo.png` due to the presence of the "Woo" in the picture.